impl/maven-impl/src/test/java/org/apache/maven/impl/DefaultModelVersionParserTest.java

```diff
@@ -22,15 +22,14 @@
 import org.eclipse.aether.util.version.GenericVersionScheme;
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.assertj.core.api.Assertions.assertThat;
 
 class DefaultModelVersionParserTest {
 
     @Test
     void parseVersion() {
         Version v = new DefaultModelVersionParser(new GenericVersionScheme()).parseVersion("");
-        assertNotNull(v);
-        assertEquals("", v.toString());
+        assertThat(v).isNotNull();
+        assertThat(v.toString()).isEqualTo("");
```

Comment:
Is there any `isEmpty()` assertion ?

Reply:
> Is there any `isEmpty()` assertion ?

If not, you can create custom assertions for AssertJ very easly. I do this quite a lot at work for custom objects/classes

Reply:
might be a bug, as failed to choose: https://docs.openrewrite.org/recipes/java/testing/assertj/simplifyassertjassertion

@timtebeek

Reply:
Any change not standardized is weakened.

Simplest way would be `JUnit Jupiter best practices` and allow both.
- https://github.com/apache/maven/pull/2320

or (continuous) migrate `JUnit` asserts to `AssertJ` like seen here.

Both will cure generic `assertTrue` statements resulting in `assertThat`. Which `lib` is actually an random impl. detail.